modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -552,9 +554,14 @@ public ObjectDefinition putObjectDefinition(
 					objectDefinition.getScope());
 		}
 
+		List<ObjectAction> objectActions = ListUtil.fromArray(
```

Comment:
For these modifications, I've asked some changes on Pedro's PR. When he finish refactoring, I'll let you know

Reply:
@seltong follow this pattern: https://github.com/liferay-objects/liferay-portal/pull/2616/commits/2403f4ef6f0303bb8db511faaa9a708b6e6b16d8